Libraries/Control/Control.cpp

```diff
@@ -8,14 +8,17 @@
 */
 #include "Control.h"
 
-Control::Control(BNO *bno, SensorMap *map) {
+Control::Control(BNO *bno, SensorMap *mapa) {
   bno_ = bno;
-  map_ = map;
+  map_ = mapa;
+  pinMode(LED1, OUTPUT); 
+  pinMode(LED2, OUTPUT); 
```

Comment:
If these leds are related to victim detection, maybe they should be in the dropkit class? Or what do you think?